testutil/network/network.go

```diff
@@ -334,6 +334,40 @@ func New(t *testing.T, cfg Config) *Network {
 	return network
 }
 
+// WaitForNBlocks waits for N blocks or timeout.
+func (n *Network) WaitForNBlocks(b int64, t time.Duration) error {
```

Comment:
We can remove this if the caller knows the latest height, which we have an API for.